packages/babel-preset-react/src/index.js

```diff
@@ -8,15 +8,19 @@ import transformReactJSXSelf from "@babel/plugin-transform-react-jsx-self";
 export default declare((api, opts) => {
   api.assertVersion(7);
 
-  const pragma = opts.pragma;
-  const pragmaFrag = opts.pragmaFrag;
-  const throwIfNamespace =
-    opts.throwIfNamespace === undefined ? true : !!opts.throwIfNamespace;
+  const {
+    pragma,
+    pragmaFrag,
+    pure,
+    throwIfNamespace = true,
+    useSpread,
+    runtime = "classic",
```

Comment:
This is a breaking change from the "new JSX" PR, because that PR accepts `runtime: false` as an alias for `runtime: "classic"`.

If this PR ends up being merged in a different release, I'll have to revert this line.

Reply:
Yeah, this should be okay. No preference for whether `runtime: false` should alias to `runtime: 'classic'` as long as `runtime: 'classic'` is the default behavior